com.dhsdevelopments.Climaxima.json

```diff
@@ -0,0 +1,170 @@
+{
+    "app-id": "com.dhsdevelopments.Climaxima",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
```

Comment:
is there a reason 3.32 doesn't work here?

Reply:
No reason. In fact, it works with freedesktop, so I have changed this.